easybuild/easyconfigs/j/Java/Java-13.eb

```diff
@@ -0,0 +1,18 @@
+# Contribution from the Crick HPC team / UL HPC Team
+# uploaded by J. Sassmannshausen
+# updated to version 202
```

Comment:
I would update this comment as well to reflect the current version of this update.